x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/parser/GrammarTests.java

```diff
@@ -28,20 +29,38 @@
  */
 public class GrammarTests extends ESTestCase {
 
-    public void testGrammar() throws Exception {
+    public void testValidQueries() throws Exception {
         EqlParser parser = new EqlParser();
-        List<Tuple<String, Integer>> lines = readQueries("/grammar-queries.eql");
+        List<Tuple<String, Integer>> lines = readQueries("/valid-queries.eql");
         for (Tuple<String, Integer> line : lines) {
             String q = line.v1();
             try {
                 parser.createStatement(q);
             } catch (ParsingException pe) {
                 if (pe.getErrorMessage().startsWith("Does not know how to handle")) {
```

Comment:
With the unsupportedTest this should be removed as all queries should be valid in this method.

Reply:
Oops - I forgot the grammar is not hooked up yet to a tree; let it be for the time being.